spec/factories/seeds/participant_identity_factory.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_participant_identity, class: "ParticipantIdentity") do
+    email { Faker::Internet.unique.email }
+    external_identifier { SecureRandom.uuid }
+
+    trait :with_user do
+      association(:user, factory: :seed_user)
```

Comment:
I think user is required here as well, so might need to move it outside the trait